src/main/kotlin/no/nav/syfo/model/DialogmeldingMapper.kt

```diff
@@ -70,28 +70,28 @@ fun XMLNotat.toHenvendelseFraLegeHenvendelse(): HenvendelseFraLegeHenvendelse {
     )
 }
 
-fun CV.toTemaKode(): TemaKode {
-    return findDialogmeldingKodeverk(s, v)!!.toTypeMelding()
+fun CV.toTemaKode(): TemaKode? {
+    return findDialogmeldingKodeverk(s, v)?.toTypeMelding()
 }
 
 fun DialogmeldingKodeverk.toTypeMelding(): TemaKode {
     return TemaKode(kodeverkOID, dn, v, arenaNotatKategori, arenaNotatKode, arenaNotatTittel)
 }
 
-fun XMLNotat.toInnkallingMoterespons(): InnkallingMoterespons {
+fun XMLNotat.toInnkallingMoterespons(): InnkallingMoterespons? {
 
-    return InnkallingMoterespons(
-        temaKode = temaKodet.toTemaKode(),
+    return if (temaKodet.toTemaKode() == null) null else InnkallingMoterespons(
+        temaKode = temaKodet.toTemaKode()!!,
```

Comment:
Er kanskje mulig å slippe unna `!!` når man allerede har sjekket at den ikke er lov? Må kanskje trekke det ut i en variabel for at det skal fungere.

Reply:
Prøver det